yt/yt/tests/integration/scheduler/test_scheduler_resource_limits.py

```diff
@@ -23,6 +23,7 @@
 ###############################################################################################
 
 MEMORY_SCRIPT = """
+#!/usr/bin/env python3
```

Comment:
Is it work correctly? (I think it should be placed on the first line, not on the second)

Reply:
that worked because interpreter is called explicitly